software_training_assignment/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>software_training_assignment</name>
+  <version>0.0.0</version>
+  <description>The software_training_assignment package</description>
+
+  <maintainer email="htieu@uwaterloo.ca">htieu</maintainer>
+
+  <license>MIT</license>
+
+
+  <build_depend>message_generation</build_depend>
+
+  <exec_depend>message_runtime</exec_depend>
+  <test_depend>gtest</test_depend>
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_export_depend>roscpp</build_export_depend>
+  <build_export_depend>std_msgs</build_export_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+  <build_depend>turtlesim</build_depend>
+  <exec_depend>turtlesim</exec_depend>
+  <build_depend>actionlib</build_depend>
+  <exec_depend>actionlib</exec_depend>
+  <build_depend>actionlib_msgs</build_depend>
+  <exec_depend>actionlib_msgs</exec_depend>
```

Comment:
if its build_depend and exec_depend you can just put the depend tag, its cleaner